tests/__snapshots__/xlsx.test.ts.snap

```diff
@@ -1220,16 +1220,17 @@ Object {
         <c:plotArea>
             <!-- how the chart element is placed on the chart -->
             <c:layout/>
-            <c:barChart>
-                <c:barDir val=\\"col\\"/>
-                <c:grouping val=\\"clustered\\"/>
-                <c:overlap val=\\"-20\\"/>
-                <c:gapWidth val=\\"70\\"/>
+            <c:lineChart>
```

Comment:
seems louche, right ?

Reply:
it's actually logic given the fix. 
If you look at the related tests `multiple charts in the same sheet` for example, we were creating the 2 charts but with the same id. This was actually destroying the first one and replacing it with the second, but it would also do this for the figure so in the end we would only have 1 exported figure and chart.

For the other test (`charts in different sheets`) we wwere creating 2 charts on different sheets but with the same id - given the structure of `ChartPlugin`  we would again have only one chart existing BUT in 2 different figures. So you'(d have fig1 in sheet 1 and fig2 in sheet2 that pointed towards the same chart. Since we fix it here,  we now correctly have 2 different charts: 1 bar and 1 line